relay-config/src/config.rs

```diff
@@ -1699,63 +1734,34 @@ impl Config {
         Duration::from_secs(self.values.cache.eviction_interval.into())
     }
 
-    /// Returns `true` if the persistent envelope buffer is enabled, `false` otherwise.
-    pub fn cache_persistent_buffer_enabled(&self) -> bool {
-        self.values.cache.persistent_envelope_buffer.is_some()
-    }
-
     /// Returns the path of the buffer file if the `cache.persistent_envelope_buffer.path` is configured.
-    pub fn cache_persistent_buffer_path(&self) -> Option<PathBuf> {
+    pub fn spool_envelopes_path(&self) -> Option<PathBuf> {
```

Comment:
I don't know what best practices are, but should we panic or something if the config has `spool.envelopes` without a path? It's basically wrong config.